health-services/resource-estimation-service/src/main/resources/db/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM egovio/flyway:4.1.2
```

Comment:
@Priyanka-eGov consider moving this service to latest 2.9 LTS

Reply:
I will not be able to determine this hence not fixed as of now , will talk with Priyanka and take a action if needed.